src/lib/components/sections/navbar.svelte

```diff
@@ -53,23 +53,28 @@
         {/each}
       </ul>
 
-      {#if jsEnabled}
-        <div class="toggle-container">
-          <!-- true is dark -->
-          <AcmToggle
-            checked={$theme === AcmTheme.Dark}
-            on:toggle={(event) => ($theme = event.detail ? AcmTheme.Dark : AcmTheme.Light)}
-          >
-            <span class="dark-toggle">
-              {#if $theme === AcmTheme.Dark}
-                <LightMode />
-              {:else}
-                <DarkMode />
-              {/if}
-            </span>
-          </AcmToggle>
-        </div>
-      {/if}
+      <div class="toggle-container" class:hidden={!jsEnabled}>
+        <!-- true is dark -->
+        <AcmToggle
+          checked={$theme === AcmTheme.Dark}
+          on:toggle={(event) => ($theme = event.detail ? AcmTheme.Dark : AcmTheme.Light)}
+        >
+          <noscript>
+            <style>
+              .dark-toggle {
+                opacity: ;
```

Comment:
Opacity is set to what?

Reply:
it's supposed to be 0

Reply:
wait, WTF LMFISDFLKSDFKSDLF HOW DID THAT HAPPEN

Reply:
ok i fix